17thblog/createAccount.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+include('config/config.php');
+include('config/init.php');
+
+echoHeader("Create Account", "Create Account");
+
+//form
+$Errors = array();
+
+if(isset($_REQUEST['createAccount'])){ //
+
+	validateFormField('Username');
+	validateFormField('Password');
```

Comment:
you probably want an extra bit of validation here. What if the username is already in use? You should check for that and throw an error if it is.